packages/core/src/Services/logout.js

```diff
@@ -37,6 +37,8 @@ const doLogout = response => {
     removeCookies('affiliate_token', 'affiliate_tracking', 'onfido_token');
     SocketCache.clear();
     sessionStorage.clear();
-    endChat();
+    if (!isTestLink) {
```

Comment:
@suthesan-deriv  if I am not wrong it's a function, so we should call it `isTestLink()`

Reply:
Yep it's function 😂 😂 ...i'll add that in